### ISSUE TITLE
issue of decoding difference fps videos

### ISSUE THREAD

**Issue by pikerbright** [NVIDIA/nvvl#11]
if I use the same one video loader and decoder difference fps videos, it may hang sometimes. 

There seems to be bugs in frame count calculation in NvDecoder handle_display_  that cause we can find the right frame in sequence.

Is it support to difference fps videos?

Thanks very much

**Comment by pikerbright** [comment]
This way to fix it:
void NvDecoder::set_time_base(AVRational time_base)
{
    log_.debug() << "set_time_base " << time_base.num << " " << time_base.den << std::endl;
    time_base_.num = time_base.num;
    time_base_.den = time_base.den;
}

Call it.
vid_decoder_->set_time_base(stream->time_base);  at VideoLoader.cpp:267

**Comment by jaredcasper** [comment]
You are right, we assume all the videos are the same frame rate (all of our videos are).  Thanks for tracking this down!

I'll take a look and get a fix in soon, something like what you've done looks correct.
